doc/source/user_guide/10min.rst

```diff
@@ -16,6 +16,15 @@ Customarily, we import as follows:
    import numpy as np
    import pandas as pd
 
+Basic data structures in pandas
+------------------------------
```

Comment:
Hi @Develpy, thanks for the PR! 
I think the Doc Build is failing because of this line is a bit too short compared to the text above it 😉

Reply:
Yep, you'll have to add another ``-``